### PR TITLE
BXMSPROD-583 nightly pipelines created

### PR DESCRIPTION
**Pull request by Ginxo**
BXMSPROD-583
please squash before merging
`Send RHPAM UMB Message to QE.` and `Send RHDM UMB Message to QE.` stages added

### CODE REVIEW

File: Jenkinsfile.justprod.nightly [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    // This is the map project, variable to store the version from this project
+                    def projectVariableMap = ['kiegroup_droolsjbpm-build-bootstrap': "kieVersion", 'jboss-integration_izpack': "izpackVersion", 'jboss-integration_installer-commons': "installerCommonsVersion"]
+                    def nightlyStages = load("$WORKSPACE/nightly.stages")
+
+                    configFileProvider([configFile(fileId: "49737697-ebd6-4396-9c22-11f7714808eb", variable: 'PRODUCTION_PROJECT_LIST')]) {
+                        println "Reading project pom file"
+                        def pom = readMavenPom file: 'pom.xml'
+                        // def variableVersionsMap = ['kieVersion': pom.version]
+                        // println "'kieVersion' variable set to ${pom.version}"
+			            def variableVersionsMap = ['kieVersion': "7.30.0.Final-redhat-00002"]
```
Comment (mareknovotny) [inline Jenkinsfile.justprod.nightly]: do we want to update this version?

File: Jenkinsfile.justprod.nightly [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                        println "'kieVersion' variable set to 7.30.0.Final-redhat-00002"
```
Comment (mareknovotny) [inline Jenkinsfile.justprod.nightly]: do we want to update this version?

File: Jenkinsfile.justprod.nightly [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                        nightlyStages(SETTINGS_XML_ID, projectCollection, "rhba/nightly", projectVariableMap, variableVersionsMap)
+                    }
+
+                    def PME_BUILD_VARIABLES = env.PME_BUILD_VARIABLES.split(";").collect{ it.split("=")}.inject([:]) {map, item -> map << [(item.length == 2 ? item[0] : null): (item.length == 2 ? item[1] : null)]}
+
+                    build job: env.RHBA_PROPERTIES_GENERATOR_PATH, parameters: [
+                        [$class: 'BooleanParameterValue', name: 'IS_PROD', value: false],
+                        [$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: PME_BUILD_VARIABLES['productVersion']],
+                        [$class: 'StringParameterValue', name: 'TIME_STAMP', value: "${PME_BUILD_VARIABLES['datetimeSuffix']}"],
+                        [$class: 'StringParameterValue', name: 'ERRAI_VERSION', value: PME_BUILD_VARIABLES['erraiVersion']],
```
Comment (mareknovotny) [inline Jenkinsfile.justprod.nightly]: we could now remove that as it is standard 3rd party dependency?

File: Jenkinsfile.launchjob.nightly [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            }
+        }
+        stage('Execute RHBA properties generator') {
+            steps {
+                script {
+                    build job: 'RHBA properties generator', parameters: [
+                        [$class: 'BooleanParameterValue', name: 'IS_PROD', value: false],
+                        [$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "1"],
+                        [$class: 'StringParameterValue', name: 'TIME_STAMP', value: "2"],
+                        [$class: 'StringParameterValue', name: 'ERRAI_VERSION', value: "3"],
```
Comment (mareknovotny) [inline Jenkinsfile.launchjob.nightly]: do we want to remove this version alignment?

File: Jenkinsfile.nightly [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                        nightlyStages(SETTINGS_XML_ID, projectCollection, "rhba/nightly", projectVariableMap)
+                    }
+
+                    def PME_BUILD_VARIABLES = env.PME_BUILD_VARIABLES.split(";").collect{ it.split("=")}.inject([:]) {map, item -> map << [(item.length == 2 ? item[0] : null): (item.length == 2 ? item[1] : null)]}
+
+                    build job: env.RHBA_PROPERTIES_GENERATOR_PATH, parameters: [
+                        [$class: 'BooleanParameterValue', name: 'IS_PROD', value: false],
+                        [$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: PME_BUILD_VARIABLES['productVersion']],
+                        [$class: 'StringParameterValue', name: 'TIME_STAMP', value: "${PME_BUILD_VARIABLES['datetimeSuffix']}"],
+                        [$class: 'StringParameterValue', name: 'ERRAI_VERSION', value: PME_BUILD_VARIABLES['erraiVersion']],
```
Comment (mareknovotny) [inline Jenkinsfile.nightly]: again Errai version manipulation should be automatical, I guess

Comment (Ginxo) [reply Jenkinsfile.nightly]: file removed since it was for testing purposes

Comment (Ginxo) [reply Jenkinsfile.nightly]: these are the parameters for the "generate properties files" job. This value comes from the build itself and it's defined on the build-config.yaml file

File: Jenkinsfile.justprod.nightly (hunk 1 above)
Comment (Ginxo) [reply Jenkinsfile.justprod.nightly]: file removed since it was for testing purposes

File: Jenkinsfile.nightly (hunk 5 above)
Comment (mareknovotny) [reply Jenkinsfile.nightly]: I think properties file doesn't need this anymore

Comment (Ginxo) [reply Jenkinsfile.nightly]: removing it from here and from the nightly template. Do I remove it from prod template as well @mareknovotny?

**Pull request merged**